setup.py

```diff
@@ -16,7 +16,7 @@
 
 setuptools.setup(
     name="comet-core",
-    version="2.5.1",
+    version="2.5.2",
```

Comment:
Not to be to picky, but I think this (great) feature is enough to warrant a minor release!
_version="2.6.0"_